capa/main.py

```diff
@@ -264,6 +276,22 @@ def install_common_args(parser, wanted=None):
             help=f"select backend, {backend_help}",
         )
 
+    if "functions" in wanted:
+        parser.add_argument(
+            "--functions",
```

Comment:
I wonder if we should use a prefix for these arguments, to help indicate they all behave similarly. Perhaps `--restrict-to-functions` and `--restrict-to-processes`.

Furthermore, we *could* dynamically generate/parse these for all the scopes, rather than hard coding them here.

I'm also not sure whether or not we want to show these by default, or allow them to be hidden/expert options.

Reply:
I've moved to this naming standard.

As for filtering other scopes, I am not sure whether that is desired or not now. Nevertheless, I have kept the room for adding any other filters in the future.

As for showing or hiding them, I lean more towards showing them since I feel like filtering by process/function is general and intuitive enough. Maybe make bb/insn or thread/call filtering hidden in the future (if we choose to add them).